internal/action/sync.go

```diff
@@ -118,6 +123,7 @@ func (s *Action) sync(ctx context.Context, store string, isAutosync bool) error
 	// sync all stores (root and all mounted sub stores).
 	for _, mp := range mps {
 		if store != "" {
+			out.Printf(ctx, "🚥 Syncing with store/remote \"%v\"...", store)
```

Comment:
```suggestion
			out.Printf(ctx, "🚥 Syncing with store/remote %q...", store)
```
%q will display a double-quoted string already.